feature: added firestore, so company is submitted in db(please note*)

cocoral opened the pull request:
1. Used my firebase credentials instead of the one that's
provided in the project in order to check the db status
2. TODO: move firebase code into the backend repo..
and make it work with express.

coralhe committed:
feature: added firesore, so company is submitted in db(please note*)

1. Used my firebase credentials instead of the one that's
provided in the project in order to check the db status
2. TODO: move firebase code into the backend repo..
and make it work with express.

Kikisek commented on src/app/App.js:
```diff
@@ -3,10 +3,29 @@ import '../assets/App.css';
+  componentDidMount() {
+    // Initialize Firebase
+    var config = {
+      apiKey: "AIzaSyDSPa0f8VuxemCZG3J8Xjvax4skoA0OUnw",
```
This is exactly what should not be on Github. @cocoral, please don't use this database for anything important 😈

Kikisek commented on package.json:
```diff
@@ -7,6 +7,7 @@
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
     "date-fns": "^2.0.0-alpha.25",
+    "firebase": "^5.8.4",
```
We should probably remove this dependency once we have it working in the backend repo.

Kikisek reviewed (dismissed):
Looks good to me. Great job @cocoral!

coralhe committed:
feature: added proxy and connected to backend endpoint.

To test it:
1.checkout backend repo on a branch with /companies endpoint
2.serve BE repo on port localhost:8081(this is the default port)
3.the submit button should run

Kikisek commented on src/app/containers/CompanyForm.js:
```diff
@@ -51,7 +51,8 @@ export class CompanyForm extends Component {
 	};
 
-	handleFormSubmit = e => {
+	// This is the submit funtion the pushes data 
```
I think there is a small typo in this comment.
